INSTALL.txt

```diff
@@ -138,18 +160,18 @@ together toolchains for that option. The toolchains are available at
 https://bitbucket.org/carlkl/mingw-w64-for-python/downloads. The site.cfg
 should be configured like so:
```

Comment:
yeah, that's a bit weird. We shouldn't rename the file to ``INSTALL.rst.txt`` though I think, the whole point of the file is that ``INSTALL.txt`` is universally recognized.